Code snippets/src/main/java/use_case/notes/CreateNotesOutputData.java

```diff
@@ -2,14 +2,22 @@
 
 import main.java.entity.Notes;
 
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
 public class CreateNotesOutputData {
     private final Notes notes;
```

Comment:
Also, I am not entirely sure final keyword is required for notes and allNotes variables.